components/prism-keepalived.js

```diff
@@ -0,0 +1,32 @@
+Prism.languages.keepalived = {
+	'comment': {
+		pattern: /(^|[^"{\\])#.*/,
```

Comment:
> Comments start with '#' or '!' to the end of the  line  and  can  start
       anywhere in a line.
[Source](https://www.keepalived.org/manpage.html)

Reply:
Yes, it could be changed to `/(#|!).*/`.